tutorial/index.jade

```diff
@@ -112,67 +69,113 @@
         </appSettings>
       ```
 
-      You can find these settings in your [Twlio Account Dashboard](/user/account) and your [Authy
-      dashboard](https://dashboard.authy.com/signup).
+      Now we need our Authy production key ([sign up for Authy
+      here](https://dashboard.authy.com/signup)). When you create an Authy
+      application the production key is found on the dashboard.
+
+      ![Authy Dashboard](//s3.amazonaws.com/howtodocs/2fa-authy-dashboard.png)
+
+      Next we need to jump over to the `UserController` to configure the Authy
+      client and create an instance method to send a one-time password.
 
       **See Also**
       * [The Twilio C# Helper Library](//www.twilio.com/docs/csharp/install)
       * [The Authy.Net Client](https://github.com/davidsulpy/authy-net-client)
 
-  .step(data-title='Request verification code' data-file='AccountVerification.Web/App_Start/IdentityConfig.cs'
-        data-highlight="132-154")
+ .step(data-title='Sending a Token on Account Creation' data-file='AccountVerification.Web/App_Start/IdentityConfig.cs' data-highlight="132-154")
     :markdown
-      ## Request verification code
+      ## Sending a Token on Account Creation
 
-      The ASP.NET Identity **UserManager** is the brain for all the user management related operation and where all our magic happens.
+      Once the user has an `authyId` we can actually send a verification code
+      to that user's mobile phone.
 
-      In order to request a verification code, we extend the `ApplicationUserManager` by adding an `AuthyClient` instance. Additionally, we add a method to check that the user exists and has been registered with Authy. If they haven't, we call `RegisterUser` on the `AuthyClient` instance, and update the `AuthyUserId`.
+      When our user is created successfully, via the form we implemented, we send
+      a token to the user's mobile phone to verify their account in our
+      controller. When the code is sent we redirect to another page where the
+      user can enter the token they received, therefore completing the verification process.
 
-      Finally, we call the `SendSms` with the `AuthyClient` which requires the user's
-      `AuthyUserId` and a `true` value for the `force` parameter. This will send the verification code to the user and force it to be sent via SMS.
+      Let's check out the code for that next!
 
-  .step(data-title='Verification code confirmation' data-file='AccountVerification.Web/App_Start/IdentityConfig.cs'
-        data-highlight="113-129")
+  .step(data-title='Verifying the Code', data-file='AccountVerification.Web/App_Start/IdentityConfig.cs', data-highlight='71-89')
```

Comment:
Could you please take a look at this one? Seems like the step and the highlighted code are unrelated.